src/validation/signupValidation.js

```diff
@@ -0,0 +1,16 @@
+import Check from '../utils/validator';
+import sendResult from '../utils/sendResult';
+
+const validateSignup = (req, res, next) => {
+  const valid = [
+    new Check({ email: req }).req().str().email()
```

Comment:
@lemoissonM  for email you don't need to specify the minimum characters since email is automatically validated